lib/config-loader.js

```diff
@@ -2,7 +2,7 @@
 
 const fs = require('fs')
 const path = require('path')
-const findUp = require('find-up')
+const escalade = require('escalade/sync')
```

Comment:
Using the sync method since most of the code in config-loader is sync. Eventually all this should be async.